terraform/deployments/govuk-publishing-platform/.tflint.hcl

```diff
@@ -0,0 +1,18 @@
+plugin "aws" {
+  enabled = true
+}
+
+config {
+  varfile = [
+    "../variables/test/infrastructure.tfvars",
+  ]
+}
+
+rule "terraform_comment_syntax" { enabled = true }
+rule "terraform_deprecated_index" { enabled = true }
+rule "terraform_required_providers" { enabled = true }
+rule "terraform_standard_module_structure" { enabled = true }
+rule "terraform_typed_variables" { enabled = true }
+rule "terraform_unused_declarations" { enabled = true }
+rule "terraform_unused_required_providers" { enabled = true }
```

Comment:
Is there a way to bring this file up a level so it applies to all deployment modules?

Reply:
At the moment I'm using the existence of this file to mark which deployments need linting. I think they'll all need slightly different config (e.g. monitoring-test/grafana doesn't use the AWS provider, monitoring-test/infra is one level deeper in the filesystem than the others, so relative links to variables would break).

Something we could look at later though.